### PR TITLE
Fix spelling in introduction.mdx

### PR DESCRIPTION
**Pull request by svenheden**

### CODE REVIEW

**Comment by ryyppy** [comment]
maybe i am wrong, but i am pretty sure that our current version is spelled correctly 😄

**Comment by kevanstannard** [comment]
Hi @ryyppy I believe it's a grammar correction, might be worth another quick check?

**Comment by fhammerschmidt** [comment]
Shouldn't it say "We **have** one of the fastest compilers & build systems"? Because "we" is probably the ReScript team or community. The compiler did not write the docs.  😉

File: pages/docs/manual/latest/introduction.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ We respect TypeScript very much and think that it's a positive force in the Java
   - Is deliberately curated to be a simple subset most folks will have an easier time to use.
   - Has **no** pitfalls, aka the type system is "sound" (the types will always be correct). E.g. If a type isn't marked as nullable, its value will never lie and let through some `undefined` value silently. **ReScript code has no null/undefined errors**.
   - Is the same for everyone. No knobs, no bikeshedding opportunity.
-  - Runs extremely fast precisely thanks to its simplicity and curation. We're one of the fastest compiler & build system for JavaScript development.
+  - Runs extremely fast precisely thanks to its simplicity and curation. We're one of the fastest compiler & build systems for JavaScript development.
```
Comment (kevanstannard) [inline pages/docs/manual/latest/introduction.mdx:23]: This is minor, but with context;

Current:

> In contrast, ReScript's type system:
> * Runs extremely fast precisely thanks to its simplicity and curation. We're one of the fastest compiler & build system for JavaScript development.

Suggestion:

> In contrast, ReScript's type system:
> * Runs extremely fast precisely thanks to its simplicity and curation. **It's** one of the fastest compiler & build **systems** for JavaScript development.

**Comment by IwanKaramazow** [comment]
I'm not a native English speaker, but the grammar correction sounds ok? If you put something simple in the sentence: "we are one of the best piano **players** in the world". I would naturally assume that `piano players` is plural, so `systems` would also be plural?

**Comment by ryyppy** [comment]
It feels weird and asymmetric to me to have "compiler" in singular, but "build systems" in plural?

Maybe it would also help to call it "compiler & build system toolchains", if plural is the grammatically correct form?

**Comment by svenheden** [comment]
Good suggestion! Updated now

**Comment by ryyppy** [comment]
thanks!

**Pull request merged**